apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.UnitTests/PriorityListTests.cs

```diff
@@ -0,0 +1,235 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Google.Cloud.Spanner.V1;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public class PriorityListTests
+    {
+        [Fact]
+        public void CanAddMultipleOfSamePriority()
+        {
+            PriorityList<TestItem> priorityList = new PriorityList<TestItem>();
```

Comment:
Nit: I'd use `var` here (and below) where you're declaring and creating the same type.